shared/wallets/wallet/settings-popup/index.js

```diff
@@ -0,0 +1,162 @@
+// @flow
+import * as React from 'react'
+import {
+  Avatar,
+  Box2,
+  ClickableBox,
+  Dropdown,
+  HeaderOnMobile,
+  Icon,
+  MaybePopup,
+  Text,
+} from '../../../common-adapters'
+import {globalColors, globalMargins, globalStyles, platformStyles, styleSheetCreate} from '../../../styles'
+
+export type Props = {
+  name: string,
+  user: string,
+  type: 'default' | 'secondary',
+  currency: string,
+  currencies: Array<string>,
+  onDelete: () => void,
+  onSetDefault: () => void,
+  onEditName: () => void,
+  onCurrencyChange: (currency: string) => void,
+}
+
+const makeDropdownItems = (props: Props) => {
+  let items = [
+    <Box2 centerChildren={true} direction="vertical" key="">
+      <Text type="BodySmall" style={{textAlign: 'center', padding: globalMargins.xsmall}}>
+        Past transactions won’t be affected by this change.
+      </Text>
+    </Box2>,
+  ]
+  for (const s of props.currencies) {
+    items.push(makeDropdownItem(s, s === props.currency))
+  }
+  return items
+}
+
+const makeDropdownItem = (item: string, isSelected: boolean) => {
+  const itemStyle = {
```

Comment:
can have an `itemStyle` in the stylesheet, and a `itemSelected` style, and then below do

```
<Text type="Header" style={collapseStyles([stylesheet.itemStyle, selected && stylesheet.itemSelected])}
```

(loosely)